test/highlight/main.heex

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<%# ^ constant %>
+<html>
+<%# <- punctuation.bracket %>
+<%#  ^ punctuation.bracket %>
+  <head>
+  <%# ^ tag %>
+    <%= csrf_meta_tag() %>
+    <%# <- keyword %>
+    <%#                 ^ keyword %>
+    <link phx-track-static rel="stylesheet" src={ Routes.static_path(@conn, "/css/app.css") } />
+    <%# <- punctuation.bracket %>
+    <%# ^ tag %>
+    <%#    ^ attribute %>
+    <%#                     ^ attribute %>
+    <%#                           ^ string %>
+    <%#                                         ^ keyword %>
+    <%#                                                                                     ^ keyword %>
+    <%#                                                                                       ^ punctuation.bracket %>
+  </head>
+  <body>
+    <%= if true do %>
+    <%# <- keyword %>
+    <%#            ^ keyword %>
+      <MyComponents.modal key=value key="value" key={ @value } />
+      <%# <- punctuation.bracket %>
+      <%# ^ module %>
```

Comment:
This module highlight extends to the `.modal` as well, and works on `.form` below. What do you think about splitting up the component_name node so it has optional children for the module and/or function? Or maybe that node shouldn't be highlighted and should be injected instead? (although now that I think about that, it might not work well for the `.form` syntax :thinking:)

Reply:
https://github.com/the-mikedavis/tree-sitter-heex/compare/083d666...f490296 I just needed to throw an ugly-enough regex at it to make it work! :smile:

Reply:
> What do you think about splitting up the component_name node so it has children node(s) for the module and/or function?

That makes sense to me, I was lazy when implementing this initially and left that up to the injected Elixir parser to handle 😅

Does your new regex consider nested modules, e.g. `<MyApp.Components.modal />`?

Reply:
yep, and I should add some highlight test cases around that!